includes/wikia/parser/templatetypes/handlers/TemplateCleanup.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class TemplateCleanup {
+
+	/**
+	 * This method will remove empty sections headers from article content
+	 *
+	 * @param $parser
+	 * @param $html
+	 *
+	 * @return bool
+	 */
+	public static function doCleanup( $parser, &$html ) {
+		wfProfileIn( __METHOD__ );
+
+		if ( $html && static::shouldBeProcessed() ) {
+			$dom = HtmlHelper::createDOMDocumentFromText( $html );
+
+			$xpath = new DOMXPath( $dom );
+			$headers = [ 'h6', 'h5', 'h4', 'h3', 'h2', 'h1' ];
```

Comment:
What do you think about making this a class field? It'd be easier to see the definitions right away that way.